native/cocos/editor-support/spine/SlotData.h

```diff
@@ -76,6 +76,8 @@ class SP_API SlotData : public SpineObject {
 
     int getIndex();
 
+    int hash;
```

Comment:
Why use public member variable?

Reply:
Hack for this version, will be removed in next version.